src/ontology/ro.Makefile

```diff
@@ -75,13 +75,6 @@ grepro:
 	cat ro-diff.txt | grep /RO_ > ro-diff.txt.ro.txt
 	cat ro-diff-core.txt | grep /RO_ > ro-diff-core.txt.ro.txt
 
```

Comment:
```suggestion

# We are not using the BFO PURL here, just two manually curated BFO components.
imports/bfo_import.owl: components/bfo-axioms.owl components/bfo-axioms.owl
	$(ROBOT) merge -i components/bfo-axioms.owl -i components/bfo-axioms.owl -o $@
```

Reply:
Err, sorry, but what good does that make? This does not fix anything, _unless_ we also update once the `components/bfo-axioms.owl` to synchronize it with the current version of upstream BFO – in which case this PR becomes an implementation of “solution 2” [described here](https://github.com/oborel/obo-relations/issues/107#issuecomment-1485063040). This just turns the manually curated components into a manually curated pseudo-import.

Reply:
I’d argue it’s even worse than the current situation, but this would make it _seem_ like we are importing upstream BFO, while in reality we are still using a manually curated frozen (and outdated!) version of it.

Reply:
Sorry I misunderstood something in slack - now I got it. If I understand it correctly and purl.../bfo.owl in its current form actually more or less corresponds to a merge of bfo-axioms with bfo-classes-minimal, we should go ahead with what you had. Sorry about the confusion!

Reply:
> purl.../bfo.owl in its current form actually more or less corresponds to a merge of bfo-axioms with bfo-classes-minimal

Yes. Except that the version published by the PURL server has updated definitions, while our manually maintained copy (or not so maintained actually) still has definitions from several years ago – hence the incompatibility!

The whole point of this PR is to use whatever version of BFO is currently published by the PURL server, so that when third-party ontologies want to import both purl.obolibrary.org/obo/ro.owl and purl.obolibrary.org/obo/bfo.owl, they can do so without being bitten by the fact that RO comes with its own outdated definitions of BFO classes.